srcpkgs/nextdns/files/nextdns.conf

```diff
@@ -0,0 +1,5 @@
+# Configuration for NextDNS
+
+# NextDNS custom configuration id
+# change id with your NextDNS id
+config id
```

Comment:
I still don't like having this config file in our tree. Is there any way around this?

Reply:
CC: @rs

Reply:
What would be your recommendation?

Reply:
@Vaelatern there are two solutions:
1. remove this config file from package and tell users to create one them self
2. add runit support in nextdns, this would be quite easy task since runit is very simple init system to deal with. But some one have to do this

Reply:
If upstream does not ship with a configuration file, neither do we.

It's ok for the runit script to not work out of the box, and require further reading to get it working. People are expected to be able to read their runit scripts and upstream documentation and make things work.

Reply:
as of latest commit is not removed from conf_files or otherwise removed.

Reply:
done!

Reply:
not done
https://files.catbox.moe/m1irr6.png

Reply:
Without an example of runit and how to write a config file that runit use this software doesn't work!
So why you want to include a non working software in the void repo?

Reply:
Simply put because it's not helpful. If you want to know more, read upstream's docs on getting started. Or read the runit script and figure out what you need. Problem solved.

Example packages that use this model:

- hooktftp
- tgt
- autox
- nomad
- consul
- seaweedfs
- netauth

are all packages we have where you really should go read upstream's documentation on how to get started. This is a good thing because it means there is less effort necessary to maintain these packages. Lower effort per package means more can be done with our lives, and that's a good thing.